labs/lab3/chapter3.md

```diff
@@ -150,7 +150,6 @@ Now we'll create the Wordpress Dockerfile. (As before, there is a reference file
         COPY latest.tar.gz /latest.tar.gz
         RUN tar xvzf /latest.tar.gz -C /var/www/html --strip-components=1 && \
             rm /latest.tar.gz && \
-            usermod -u 27 apache && \
```

Comment:
this seems risky.. is this fixing something? if not then I say if it ain't broke, don't fix it.

Reply:
i added this some time last year when moving away from running the containers as root. at that time we had to chown the host dirs in order to properly access them within the container as the mysql/apache users.  this dockerfile line was added to align the uid for apache w/ mysql's uid for the sake of simplicity in our lab.
https://github.com/tchughesiv/rh-container-lab/blob/master/labs/lab3/chapter3.md#build-images-test-and-push

so, in reality, this line didn't exist at last year's summit.

did you have issues running through it? i mean, it won't hurt anything to keep it in but it has no purpose at this point since we're not chowning the host dirs.

Reply:
ok, just checking

Reply:
ya, non-root added some dockerfile steps and w/ podman im trying to clean some of them up where it makes sense. it appears we no longer have to worry about chowning the host dirs at all... but would love additional checks.

Reply:
Before we merge this, let me look at this closer and run some additional tests to be absolutely sure. It may function but I want to look at the mount from within the container